source/locale/en/symbols.dic

```diff
@@ -79,7 +79,7 @@ _	line	most
 }	right brace	most
 |	bar	most
 ¦	broken bar	most
-~	tilda	most
+~	tild	most
```

Comment:
I believe this spelling is intentional. In English, this character is spelt `tilde` not `tild`. However, it is pronounced `til-da`, hence the current spelling

Reply:
Ok, this is what I found when I checked and compared the symbol table in my native language. I'm sorry for not thinking that much about it, I've turned this PR off.